src/main/java/nl/tudelft/dnainator/ui/widgets/animations/SlidingAnimation.java

```diff
@@ -19,18 +18,50 @@
 	 * @param pane         The {@link Pane} to be animated.
 	 * @param size         The size over which the animation will occur.
 	 * @param duration     The duration of the animation.
+	 * @param pos          The position of the {@link Pane}.
 	 */
-	public SlidingAnimation(Pane pane, double size, double duration) {
-		super(pane, size, duration);
+	public SlidingAnimation(Pane pane, double size, double duration, Position pos) {
+		super(pane, size, duration, pos);
 	}
 	
 	@Override
 	public void setupAnimation() {
-		da = this;
-		curSize = size;
+		newSize = size;
 		setCycleDuration(Duration.millis(duration));
 	}
 	
+	@Override
+	protected void interpolate(double frac) {
+		setCurSize(frac);
+		setMovement(newSize);
+	}
+	
+	/**
+	 * Set the newly computed {@link Pane} size.
+	 * @param frac a fraction (that goes from 0.0 to 1.0).
+	 */
+	public abstract void setCurSize(double frac);
+	
+	/**
+	 * Does the sliding animation, through translations and width changing of the {@link Pane}.
+	 * @param move the size of the movement.
+	 */
+	public void setMovement(double move) {
```

Comment:
Should probably have caught this earlier, but this parameter is unused. All variables used in this function are instance variables.